django/utils/glob.py

```diff
@@ -1,19 +1,2 @@
-import os.path
-import re
-
-from django.utils import six
-
-# backport of Python 3.4's glob.escape
-
-if six.PY3:
-    from glob import escape as glob_escape
-else:
-    _magic_check = re.compile('([*?[])')
-
-    def glob_escape(pathname):
-        """
-        Escape all special characters.
-        """
-        drive, pathname = os.path.splitdrive(pathname)
-        pathname = _magic_check.sub(r'[\1]', pathname)
-        return drive + pathname
+# To be deprecated
```

Comment:
I think that could be removed without deprecation since it's undocumented and only used once in Djagno.